curriculum/challenges/english/06-information-security-and-quality-assurance/information-security-with-helmetjs/configure-helmet-using-the-parent-helmet-middleware.english.md

```diff
@@ -23,7 +23,7 @@ app.use(helmet({<br>
 &nbsp;&nbsp;dnsPrefetchControl: false     // disable<br>
 }))
 </blockquote>
-We introduced each middleware separately for teaching purpose, and for ease of testing. Using the ‘parent’ helmet() middleware is easiest, and cleaner, for a real project.
+We introduced each middleware separately for teaching purpose, and for ease of testing. Using the ‘parent’ <code>helmet()</code> middleware is easiest, and cleaner, for a real project.
```

Comment:
Since this line is being modified, let's go ahead and clean up the verbiage.  Here is my suggestion, though there could be better options.
```suggestion
We introduced each middleware separately for teaching purposes and for ease of testing. Using the ‘parent’ <code>helmet()</code> middleware is an easy and cleaner way for a real project.
```